kube-webhook-certgen.yaml

```
@@ -0,0 +1,34 @@
+package:
+  name: kube-webhook-certgen
+  version: 1.5.2
+  epoch: 0
+  description: Tools to help with self signed cert generation for Kubernetes test environment
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    repositories:
+      - https://dl-cdn.alpinelinux.org/alpine/edge/main
+      - https://dl-cdn.alpinelinux.org/alpine/edge/community
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: cf887ac4a6190e91a053da8364578659cb03a1bf
+      repository: https://github.com/jet/kube-webhook-certgen
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      ldflags: -s -w -X 'github.com/jet/kube-webhook-certgen/core.Version=v${{package.version}}'
+      output: kube-webhook-certgen
+      packages: .
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: jet/kube-webhook-certgen
+    strip-prefix: v
```

```suggestion
package:
  name: kube-webhook-certgen
  version: 1.5.2
  epoch: 0
  description: Tools to help with self signed cert generation for Kubernetes test environment
  copyright:
    - license: Apache-2.0

environment:
  contents:
    packages:
      - build-base
      - busybox
      - ca-certificates-bundle
      - go

pipeline:
  - uses: git-checkout
    with:
      expected-commit: cf887ac4a6190e91a053da8364578659cb03a1bf
      repository: https://github.com/jet/kube-webhook-certgen
      tag: v${{package.version}}

  - uses: go/build
    with:
      ldflags: -s -w -X 'github.com/jet/kube-webhook-certgen/core.Version=v${{package.version}}'
      output: kube-webhook-certgen
      packages: .

  - uses: strip

update:
  enabled: true
  github:
    identifier: jet/kube-webhook-certgen
    strip-prefix: v

```
Here we use the Wolfi repository packages by default(can't work simultaneously with Alpine packages), you will just have to include the packages required.

Looks like the upstream repository is not being maintained anymore(the last commit was 3 years ago). so there is a lot of CVE